ansible/istio/tasks/change_scc.yml

```diff
@@ -6,8 +6,4 @@
     - istio-egress-service-account
     - istio-pilot-service-account
     - istio-mixer-service-account
-    - istio-ca-service-account
```

Comment:
Why do we have to remoe "istio-ca-service-account" @geoand ?

Reply:
We don't actually... My local code has it. I'm checking to see why it appears to be deleted

Reply:
I think that it was part of a previous distro of istio 0.3.0 or 0.2.12 ....

Reply:
https://github.com/geoand/istio-integration/blob/scc-updates/ansible/istio/tasks/change_scc.yml

Reply:
It looks like a bug in GitHubs's UI

Reply:
I definitely haven't deleted it :)